common/src/test/java/com/revenuecat/purchases/backend_integration_tests/LoadShedderBackendIntegrationTest.kt

```diff
@@ -0,0 +1,52 @@
+package com.revenuecat.purchases.backend_integration_tests
+
+import com.revenuecat.purchases.PurchasesError
+import com.revenuecat.purchases.common.networking.Endpoint
+import com.revenuecat.purchases.common.offlineentitlements.ProductEntitlementMapping
+import io.mockk.verify
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Test
+
+class LoadShedderBackendIntegrationTest: BaseBackendIntegrationTest() {
```

Comment:
The way I structured this right now, these and production backend integration tests live in the same place and are run at the same time. They are pretty fast though so I think that's fine. Lmk if you would prefer to run them separately.